stl/inc/yvals_core.h

```diff
@@ -470,6 +471,13 @@
 #define _CONSTEXPR20 inline
 #endif // ^^^ inline (not constexpr) in C++17 and earlier ^^^
 
+// Functions that became constexpr in C++20, and requires is_constant_evaluated
+#ifdef __cpp_lib_is_constant_evaluated
+#define _CONSTEXPR20_ICE constexpr
+#else // ^^^ constexpr witn is_constant_evaluated / vvv inline (not constexpr) without is_constant_evaluated vvv

```

Comment:
typo `witn` and outdated `inline` in the comment.

Reply:
Thanks!